src/steps/ms-defender/machine/index.test.ts

```diff
@@ -19,3 +19,15 @@ test('#fetchMachines', async () => {
   expect(stepResult).toMatchStepMetadata(stepConfig);
   expect(stepResult).toMatchSnapshot();
 });
+
+test('#fetchEndpoints', async () => {
+  recording = setupProjectRecording({
+    directory: __dirname,
+    name: '#fetchEndpoints',
+  });
+
+  const stepConfig = buildStepTestConfigForStep(Steps.FETCH_ENDPOINTS.id);
+  const stepResult = await executeStepWithDependencies(stepConfig);
+  expect(stepResult).toMatchStepMetadata(stepConfig);
+  expect(stepResult).toMatchSnapshot();
+}, 5000000);
```

Comment:
```suggestion
}, 5000000);
```
I'm not that familiar with jest, but I'm guessing this is the test timeout? If so could we reduce it a little bit. I don't know if this test hangs or not, but I wouldn't want it to run for an hour.

Reply:
Hey, yhese timeouts allow us (at times) to make recordings, but later if you were to just run the tests against stored recordings/snapshots, they are ignored. We usually remove them after the recordings are made, but some stick around.

I've removed the timeouts currently :+1: